src/algebraic_topology/dold_kan/compatibility.lean

```diff
@@ -176,6 +176,119 @@ end
 
 lemma equivalence_functor : (equivalence hF hG).functor = F ⋙ eB.inverse := rfl
 
+omit hG hF
+
+/-- The isomorphism `eB.functor ⋙ e'.inverse ⋙ e'.functor ≅ eB.functor` deduced
+from the counit isomorphism of `e'`. -/
+@[simps hom_app]
+def τ₀ : eB.functor ⋙ e'.inverse ⋙ e'.functor ≅ eB.functor :=
```

Comment:
The name `τ₀` is not great, unless it is standard for what you are doing. It is however tucked away in a namespace so perhaps wouldn't cause too much trouble. Will you be using this declaration outside this file? If not, it could be private; if so it could be called some horrible long mathlib name, or you could just leave it because in my mind it's very much a "lemma 3" kind of definition (needs to be there, is really called "this square obviously commutes" or whatever). Same comments for tau_1 and `υ` later on.

Reply:
Actually, these `τ₀` etc appear in the assumptions to the constructions and statements in this file. They are used for the Dold-Kan equivalence for pseudoabelian categories (which is the very reason for the stuff in this file `compatibility.lean`). Then, keeping these in their separate namespace looks like the best solution.